models/categories.go

```diff
@@ -0,0 +1,78 @@
+package models
+
+import (
+	"context"
+	"errors"
+	"strconv"
+	"time"
+
+	"github.com/Daniorocket/cookit-backend/lib"
+	"github.com/globalsign/mgo/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var CollectionCategory = "categories"
+var ErrCreateCategory = errors.New("Failed to create category record")
+var ErrFindCategory = errors.New("Failed to find category record")
+
+type Category struct {
+	ID      string   `json:"id" bson:"id"`
+	LabelPL string   `json:"labelPL" bson:"label_pl"`
+	LabelEN string   `json:"labelEN" bson:"label_en"`
+	File    lib.File `json:"file" bson:"file"`
+}
+
+func GetAllCategories(client *mongo.Client, db, page, limit string) ([]Category, int64, error) {
+	var category Category
+	categories := []Category{}
+	p, err := strconv.Atoi(page)
+	if err != nil {
+		return nil, 0, err
+	}
+	l, err := strconv.Atoi(limit)
+	if err != nil {
+		return nil, 0, err
+	}
```

Comment:
Wydaje mi się, że GetAllCategories powinno przyjmować już inty, a nie bawić się w Atoi. Handler w tym przypadku powinien być od obródki danych, a to powinno je przetwarzać tylko

Reply:
Tutaj też nie wiedziałem, w którym miejscu zrobić rzutowanie, więc zrobiłem to intuicyjnie, poprawiam